vilebot/src/main/java/com/oldterns/vilebot/util/NewsParser.java

```diff
@@ -19,12 +25,6 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import java.io.IOException;
-import java.net.URL;
-import java.util.LinkedHashMap;
-import java.util.List;
-import java.util.regex.Matcher;
-
 public abstract class NewsParser
```

Comment:
Now that this class is used for something other than news parsing, maybe rename it to something more generic like `FeedParser`? Not a blocker, just a recommendation.